### PR TITLE
Updated gitignore

### PR DESCRIPTION
**Pull request by daskepon**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | No
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A

## Summary
I updated .gitignore.

## Detail
- added coverage.xml in case `phpunit --coverage-clover=coverage.xml` is executed
- removed .php_cs.cache, because this project use styleci now
- added .DS_Store for macOS developpers

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 /vendor/
 /test/fixture/recipe/tmp/
-.php_cs.cache
 *.phar
 phpunit.xml
+coverage.xml
+.DS_Store
```
Comment (antonmedv) [inline .gitignore]: This is os specific ignore. It should go into your system global ignore.

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
 ### Changed
 - Laravel recipe should not run `artisan:cache:clear` in `deploy` task
 - Pass-through the quiet mode into the git commands for updating code
+- Updated .gitignore
```
Comment (antonmedv) [inline CHANGELOG.md]: I think this is chore update, no users to inform.

File: .gitignore (hunk 1 above)
Comment (daskepon) [reply .gitignore]: OK. I'll revert this change. Thank you.

File: CHANGELOG.md (hunk 2 above)
Comment (daskepon) [reply CHANGELOG.md]: OK. I'll revert this change. Thank you.

**Comment by daskepon** [comment]
I Changed this PR.

**Pull request merged**